integration_tests/sdk/conftest.py

```diff
@@ -1,17 +1,20 @@
-import os
-
 import pytest
 from aqueduct.constants.enums import ServiceType
 from aqueduct.models.dag import DAG, Metadata
 
 from aqueduct import Client, globals
+from sdk.setup_integration import (
+    get_aqueduct_config,
+    list_data_integrations,
+    setup_data_integrations,
+)
 from sdk.shared import globals as test_globals
 from sdk.shared.utils import delete_flow, generate_new_flow_name
 from sdk.shared.validator import Validator
 
 
 def pytest_addoption(parser):
-    parser.addoption(f"--data", action="store", default="aqueduct_demo")
+    parser.addoption(f"--data", action="store", default=None)
```

Comment:
why we removed the default here?

Reply:
I think I understood now, that this flag now have controls specific data integration to use so we should follow test config file if not specified.

Reply:
yep, exactly :)